enterprise/api_client/discovery.py

```diff
@@ -72,6 +79,36 @@ def __init__(self, user):
         self.user = user
         self.client = course_discovery_api_client(user)
 
+    def get_search_results(self, querystring=None):
+        """
+        Return a search result from all data.
+
+        Returns:
+            dict: API response with search results
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+        )
+
+    def get_paginated_search_results(self, querystring=None):
+        """
+        Return paginated search result from all data.
+
+        Returns:
+            dict: API response with search results, as well as links to next and previous pages.
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+            traverse_pagination=False,
```

Comment:
@UmanShahzad Thanks for digging that up! The docstring doesn't do the best job of explaining the term IMHO, but your explanation matches my assumptions :)